### PR TITLE
Add sphinxcontrib-confluencebuilder

### PR DESCRIPTION
**Pull request by maresb**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maresb** [comment]
@conda-forge/help-python ready for review!

File: recipes/sphinxcontrib-confluencebuilder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - pip check
+    - sphinx-build-confluence --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sphinx-contrib/confluencebuilder
+  summary: Sphinx extension to output Atlassian Confluence Storage Markup documents and publish to Confluence instances.
+  license: BSD-3-Clause
```
Comment (dopplershift) [inline recipes/sphinxcontrib-confluencebuilder/meta.yaml:43]:
```suggestion
  license: BSD-2-Clause
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor correction to the license and this is good to go in.

File: recipes/sphinxcontrib-confluencebuilder/meta.yaml (hunk 1 above)
Comment (maresb) [reply recipes/sphinxcontrib-confluencebuilder/meta.yaml]: Thanks! Looks like there may be a bug in grayskull...

**Review by dopplershift — APPROVED** [review]

**Pull request merged**